### PR TITLE
Bugfixes for the upcoming async feature

### PR DESCRIPTION
**Pull request by jmunhoz**
It adds some missing installation dependencies related to required packages, minimal python version avoiding default one and right 1-to-1 behaviour for private messages when several 1-to-1 DMs are in place with the bot.

### CODE REVIEW

File: matrixbot/matrix.py [hunk 1]
```diff
@@ -257,6 +257,11 @@ def send_notice(self, room_id, message):
                              room_id, message)
 
     def send_private_message(self, user_id, message, room_id=None):
+        # Just add a first case: if the channel is 1-to-1 then reply directly using this channel
```
Comment (psaavedra) [inline matrixbot/matrix.py:260]: Format: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

File: setup.py [hunk 2]
```diff
@@ -64,6 +64,7 @@ def exec_file(path_segments):
         "pytz==2018.9",
         "requests==2.21.0",
         "python-dateutil==2.8.1",
+        "asyncio==3.4.3",
```
Comment (psaavedra) [inline setup.py:67]: Not needed. The asyncio module is part of the Python standard library since Python 3.4

**Review by psaavedra — CHANGES_REQUESTED** [review]

File: matrixbot/matrix.py (hunk 1 above)
Comment (jmunhoz) [reply matrixbot/matrix.py]: Done.

File: setup.py (hunk 2 above)
Comment (jmunhoz) [reply setup.py]: Done.

**Comment by psaavedra** [comment]
thanks for the PR

File: matrixbot/matrix.py [hunk 3]
```diff
@@ -257,6 +257,12 @@ def send_notice(self, room_id, message):
                              room_id, message)
 
     def send_private_message(self, user_id, message, room_id=None):
+        # Just add a first case: if the channel is 1-to-1 then
```
Comment (psaavedra) [inline matrixbot/matrix.py:260]: This fixes https://github.com/psaavedra/matrix-bot/issues/36

**Pull request merged**